docs/DeveloperGuide.md

```diff
@@ -961,6 +961,16 @@ For all use cases below, the **System** is the `Wishful Shrinking` and the **Act
   1. Wishful Shrinking shows all valid commands.
 
      Use case ends.
+**Use case: Select a recipe in the recipe list**
+
+**MSS**
+
+  1. User requests to view a single recipe.
```

Comment:
i think can add extension when index is not correct?

Reply:
Agree.